code/decals/decals.cpp

```diff
@@ -270,6 +273,8 @@ struct Decal {
 			if (smi->blown_off) {
 				return false;
 			}
+		} else {
+			Assertion(false, "Only ships are currently supported for decals!");
```

Comment:
Shouldn't we return `false` here as well? What will happen in release builds without it?

Reply:
Good point